tiny_hippo_run.py

```diff
@@ -33,7 +34,8 @@ def main():
     Main loop of the program, does the following
     1. Validates the users given mac addresses
     2. Runs system privacy checks
-    3. Sniffs packets on "wlan0" and analyzes the packet against signatures and privacy rules
+    3. Runs scanning analysis of the IoT devices
```

Comment:
Just a general concern comment: should we always do the passive scan when we execute tiny_hippo_run.py? I am just unsure if this is the proper way to go about doing a passive scan. You would only need to scan the devices once to check for open ports and whatnot. It is not like the IoT devices are going to magically open or close ports overnight.

Reply:
Surprisingly, the number of open ports changed frequently between runs, zero idea why. I ran normal nmap too and got the same results

Reply:
That......that is super weird. I am not sure what to make of that. Kinda concerning